src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/MapAlignmentAlgorithmIdentification.h

```diff
@@ -206,6 +213,22 @@ namespace OpenMS
     template <typename MapType>
     bool getRetentionTimes_(MapType& features, SeqToList& rt_data)
     {
+      if (param_.getValue("score_cutoff") == "false")
```

Comment:
is this new parameter documented?

Reply:
yes, see MapAlignmentAlgorithmIdentification.cpp

@jpfeuffer I suggest to introduce a member variable `bool score_cutoff_` because calling `getValue` is very, very slow!

Reply:
true. but only will be called once per map. So usually only once per application.

Reply:
done. for use_feature_rt also